pynxtools/eln_mapper/schema_eln.py

```diff
@@ -0,0 +1,282 @@
+"""This module intended to generate schema eln which usually randeredto NOMAD."""
+
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import Dict, Any
+import xml.etree.ElementTree as ET
+import yaml
+from pynxtools.eln_mapper.eln import retrieve_nxdl_file
+from pynxtools.dataconverter.helpers import remove_namespace_from_tag
+
+
+NEXUS_TYPE_TO_NOMAD_TYPE = {'NX_CHAR': {'convert_typ': 'str',
```

Comment:
better rename to NEXUS_TYPE_TO_NUMPY_TYPE rather than NOMAD_TYPE because an np.float64 etc. is nothing specific for NOMAD